plugins/services/src/js/stores/MarathonStore.js

```diff
@@ -472,8 +472,8 @@ class MarathonStore extends GetSetBaseStore {
     const groups = new ServiceTree(data);
 
     const apps = groups.reduceItems(function(map, item) {
```

Comment:
Let's 🔥  `CompositeState` and then remove this bit instead of investing any time trying to document a workaround that is only there to support old legacy code.